app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -204,4 +212,51 @@
             {{t 'institutions.dashboard.users_list.empty'}}
         </list.empty>
     </PaginatedList::HasMany>
+    <OsfDialog @isOpen={{this.messageModalShown}} @onClose={{this.toggleMessageModal}} as |dialog|>
+        <dialog.heading>
+            {{t 'institutions.dashboard.send_message_modal.title'}}
+        </dialog.heading>
+        <dialog.main>
+            <div>
+                <label for='message-text' local-class='message-label'>
+                    {{t 'institutions.dashboard.send_message_modal.opening_message_label'}}
+                </label>
+                <Textarea
+                  id='message-text'
+                  local-class='message-textarea'
+                  @value={{this.messageText}}
+                />
+                <div>
+                    {{t 'institutions.dashboard.send_message_modal.closing_message_label' adminName=this.currentUser.user.fullName htmlSafe=true}}
+                </div>
+                <div local-class='checkbox-container'>
+                    <label local-class='checkbox-item'>
+                        <input type='checkbox' checked={{this.cc}} {{on 'change' this.toggleCc}} />
+                        {{t 'institutions.dashboard.send_message_modal.cc_label'}}
+                    </label>
+                    <label local-class='checkbox-item'>
+                        <input type='checkbox' checked={{this.replyTo}} {{on 'change' this.toggleReplyTo}} />
+                        {{t 'institutions.dashboard.send_message_modal.reply_to_label'}}
+                    </label>
+                </div>
+            </div>
+        </dialog.main>
+        <dialog.footer>
+            <Button
+                @type='secondary'
+                {{on 'click' this.toggleMessageModal}}
+            >
+                {{t 'general.cancel'}}
+            </Button>
+            <Button
+                @type='primary'
+                @disabled={{not this.messageText.trim}}
+                {{on 'click' (queue (perform this.sendMessage))}}
+                {{on 'click' this.toggleMessageModal}}
```

Comment:
You can combine these into one line like `{{on 'click' (queue (perform this.sendMessage) this.toggleMessageModal)}}`